question5.py

```diff
@@ -0,0 +1,11 @@
+numbers = [9,4,8,10,2,4,8,3,14,4,8]
+
+num = 12
+
+pairs = []
+
+for i in range(len(numbers)):
+    for j in range(len(numbers)):
```

Comment:
Can't I do without using index? Like accessing elements directly from numbers list in loop?